Resources/doc/deployment/deployment.md

```diff
@@ -1,133 +1,96 @@
 # Using deployment
 
-We use Capifony as a base, but we extended it with our own specific
-configuration for usage with the framework.
+## Install deployer
 
-All configuration for the specific stage (staging, production) is done in the
-`deployments/stages/`-folder, wherein each stage has its own file.
+To use the deployer we need to have a local installation of the deployer.
```

Comment:
"a local installation of Deployer"

May be also provide a link to the Deployer website.

Reply:
771ee017859d1412017ef62ece84d5c6a39d6be6